Story/Shattersword.cs

```diff
@@ -0,0 +1,124 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class Shattersword
```

Comment:
Since this is a memberlocked map u should add [Member] in file naming as a suffix (not in class name) also included in member check in 0Allstoires